test/syscalls/linux/tuntap.cc

```diff
@@ -490,5 +491,36 @@ TEST_F(TuntapTest, WriteHangBug155928773) {
   write(sock, "hello", 5);
 }
 
+// EmptyInterfaceName tests empty interface name at creation of tun/tap.
+TEST_F(TuntapTest, EmptyInterfaceName) {
+  SKIP_IF(!ASSERT_NO_ERRNO_AND_VALUE(HaveCapability(CAP_NET_ADMIN)));
+
+  FileDescriptor sender = ASSERT_NO_ERRNO_AND_VALUE(
+      Socket(AF_INET, SOCK_STREAM | SOCK_NONBLOCK, IPPROTO_TCP));
+
+  char* empty_name = "";
+  struct ifreq ifr_set = {};
+
+  // Interface creation.
+  ASSIGN_OR_RETURN_ERRNO(FileDescriptor fd, Open(kDevNetTun, O_RDWR));
+
+  std::memset(&ifr_set, 0x0, sizeof(ifr_set));
+  ifr_set.ifr_flags = IFF_TAP;
+  strncpy(ifr_set.ifr_name, dev_name.c_str(), IFNAMSIZ);
```

Comment:
What is `dev_name` here? IIUC `ifr_set.ifr_name` should be "".

Same below.

Reply:
I assume it was supposed to be empty_name. As Kevin says, it can probably just be "".

Reply:
Fixed